docs/project/contribution_tools.md

```diff
@@ -224,6 +225,15 @@ Dwarf Error: DW_FORM_strx1 found in non-DWO CU
 It means that the version of GDB used is too old, and does not support the DWARF
 v5 format.
 
+### System Development Libraries (zlib, etc.)
```

Comment:
```suggestion
### zlib (Linux-only)
```

Reply:
Isn’t that the opposite what Chandlerc previously requested?https://github.com/carbon-language/carbon-lang/pull/1558#discussion_r927910578

Reply:
I don't feel super strongly. It seemed like we should just have a section for system libraries because I suspect this won't be the last, but if Jon would prefer to wait I guess we can.

Reply:
@jonmeow personally I'd rather have a general section but if you confirm I'll change it back.

Reply:
Right now it states there are multiple "(zlib, etc)". However, there's only one. I understand the desire to be generic, but I think it'll spur more questions about why we aren't mentioning others.